### DISCUSSION TITLE
its free to use?

### DISCUSSION THREAD

**Discussion by ivan-vilches** [illacloud/illa-builder#594] [General]
its free to use? if yes it have limitations ?
thanks you

**Comment by cyx0928** [comment]
Hey developer:
Yes, ILLA is free to use, users can use self-host to access all the features. We will provide cloud service soon at the end of Dec. You can join our Discord to win free access codes. The only difference between cloud service and local host is that you save a lot of time in deployment.

Best,

ILLA team
